We need to use gcc here because we don't have cc.

--- libvncserver-LibVNCServer-0.9.14/test/includetest.sh.orig	2023-12-04 08:57:31.644556391 +0100
+++ libvncserver-LibVNCServer-0.9.14/test/includetest.sh	2023-12-04 08:57:44.067316854 +0100
@@ -22,4 +22,4 @@
 }
 " > "$TMPDIR"/includetest.c
 
-cc -I "$TMPDIR/$PREFIX" "$TMPDIR"/includetest.c
+gcc -I "$TMPDIR/$PREFIX" "$TMPDIR"/includetest.c
